Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -210,6 +213,7 @@ + (BranchActivityItemProvider *)getBranchActivityItemWithParams:(NSDictionary *)
 
 static BOOL bnc_useTestBranchKey = NO;
 static NSString *bnc_branchKey = nil;
+static BOOL bnc_disableFingerprintIDInCrashlyticsReports = NO;
```

Comment:
That's fine with me. I wonder if this is necessary though. If an app has AdSupport.framework linked, and a user has ad tracking enabled, then Fabric will definitely have their IDFA once the app is launched. There may be cases where this switch is desirable. I'll make this change, but we can probably also consider removing now or deprecating and removing later.